entrypoint.sh

```diff
@@ -44,6 +44,7 @@ if [[ -n "$working_directory" ]]; then
 fi
 
 if [[ -n "$glob_root_file" ]]; then
+  shopt -s extglob
```

Comment:
Could we simply enable this flag globally?

Reply:
Yes absolutely, I was afraid of breaking stuff by enabling the flag globally but it shouldn't